```
======================================================================
FAIL: test_case_reindexer_v2 (testapps.test_pillowtop.tests.test_reindexer.PillowtopReindexerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 201, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_reindexer.py", line 74, in test_case_reindexer_v2
    self._assert_case_is_in_es(case)
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_reindexer.py", line 123, in _assert_case_is_in_es
    self.assertEqual(1, results.total)
AssertionError: 1 != 0
```

--- a/corehq/apps/hqcase/management/commands/ptop_reindexer_v2.py
+++ b/corehq/apps/hqcase/management/commands/ptop_reindexer_v2.py
@@ -4,7 +4,7 @@ from django.core.management import BaseCommand, CommandError
 
 from corehq.pillows.application import get_app_reindexer
 from corehq.pillows.case import (
-    get_couch_case_reindexer, get_sql_case_reindexer, get_resumable_couch_case_reindexer
+    get_couch_case_reindexer, get_sql_case_reindexer
 )
 from corehq.pillows.case_search import get_case_search_reindexer
 from corehq.pillows.domain import get_domain_reindexer
@@ -61,7 +61,6 @@ class Command(BaseCommand):
             'group': get_group_reindexer,
             'groups-to-user': get_groups_to_user_reindexer,
             'case': get_couch_case_reindexer,
-            'resumable-case': get_resumable_couch_case_reindexer,
             'form': get_couch_form_reindexer,
             'sql-case': get_sql_case_reindexer,
             'sql-form': get_sql_form_reindexer,

--- a/corehq/pillows/case.py
+++ b/corehq/pillows/case.py
@@ -113,24 +113,12 @@ def get_couch_case_to_elasticsearch_pillow(pillow_id='CouchCaseToElasticsearchPi
 
 
 def get_couch_case_reindexer():
-    return get_default_reindexer_for_elastic_pillow(
-        pillow=CasePillow(online=False),
-        change_provider=CouchViewChangeProvider(
-            couch_db=CommCareCase.get_db(),
-            view_name='cases_by_owner/view',
-            view_kwargs={
-                'include_docs': True,
-            }
-        )
-    )
-
-
-def get_resumable_couch_case_reindexer():
     return ResumableBulkElasticPillowReindexer(
-        pillow=CasePillow(online=False),
+        name="CouchCaseToElasticsearchPillow",
         doc_types=[CommCareCase],
         elasticsearch=get_es_new(),
-        index_info=_get_case_index_info()
+        index_info=_get_case_index_info(),
+        doc_transform=transform_case_for_elasticsearch
     )
 
 

--- a/corehq/pillows/case.py
+++ b/corehq/pillows/case.py
@@ -1,24 +1,22 @@
 import copy
 import datetime
+import logging
 
 from casexml.apps.case.models import CommCareCase
 from corehq.apps.change_feed import topics
 from corehq.apps.change_feed.consumer.feed import KafkaChangeFeed
 from corehq.elastic import get_es_new
 from corehq.form_processor.change_providers import SqlCaseChangeProvider
-from corehq.pillows.mappings.case_mapping import CASE_MAPPING, CASE_INDEX, CASE_ES_TYPE, CASE_ES_ALIAS
+from corehq.pillows.mappings.case_mapping import CASE_INDEX_INFO
 from corehq.pillows.utils import get_user_type
 from dimagi.utils.couch import LockManager
-from .base import HQPillow
-import logging
 from pillowtop.checkpoints.manager import PillowCheckpoint, PillowCheckpointEventHandler
-from pillowtop.es_utils import doc_exists, ElasticsearchIndexInfo, get_index_info_from_pillow
+from pillowtop.es_utils import doc_exists
 from pillowtop.listener import lock_manager
 from pillowtop.pillow.interface import ConstructedPillow
 from pillowtop.processors.elastic import ElasticProcessor
-from pillowtop.reindexer.change_providers.couch import CouchViewChangeProvider
-from pillowtop.reindexer.reindexer import get_default_reindexer_for_elastic_pillow, \
-    ElasticPillowReindexer, ResumableBulkElasticPillowReindexer
+from pillowtop.reindexer.reindexer import ElasticPillowReindexer, ResumableBulkElasticPillowReindexer
+from .base import HQPillow
 
 UNKNOWN_DOMAIN = "__nodomain__"
 UNKNOWN_TYPE = "__notype__"
@@ -34,11 +32,11 @@ class CasePillow(HQPillow):
     """
     document_class = CommCareCase
     couch_filter = "case/casedocs"
-    es_alias = CASE_ES_ALIAS
-    es_type = CASE_ES_TYPE
+    es_alias = CASE_INDEX_INFO.alias
+    es_type = CASE_INDEX_INFO.type
 
-    es_index = CASE_INDEX
-    default_mapping = CASE_MAPPING
+    es_index = CASE_INDEX_INFO.index
+    default_mapping = CASE_INDEX_INFO.mapping
 
     @classmethod
     def get_unique_id(cls):
@@ -78,7 +76,7 @@ def get_sql_case_to_elasticsearch_pillow(pillow_id='SqlCaseToElasticsearchPillow
     )
     case_processor = ElasticProcessor(
         elasticsearch=get_es_new(),
-        index_info=ElasticsearchIndexInfo(index=CASE_INDEX, type=CASE_ES_TYPE),
+        index_info=CASE_INDEX_INFO,
         doc_prep_fn=transform_case_for_elasticsearch
     )
     return ConstructedPillow(
@@ -98,7 +96,7 @@ def get_couch_case_to_elasticsearch_pillow(pillow_id='CouchCaseToElasticsearchPi
     )
     case_processor = ElasticProcessor(
         elasticsearch=get_es_new(),
-        index_info=ElasticsearchIndexInfo(index=CASE_INDEX, type=CASE_ES_TYPE),
+        index_info=CASE_INDEX_INFO,
         doc_prep_fn=transform_case_for_elasticsearch
     )
     return ConstructedPillow(
@@ -117,7 +115,7 @@ def get_couch_case_reindexer():
         name="CouchCaseToElasticsearchPillow",
         doc_types=[CommCareCase],
         elasticsearch=get_es_new(),
-        index_info=_get_case_index_info(),
+        index_info=CASE_INDEX_INFO,
         doc_transform=transform_case_for_elasticsearch
     )
 
@@ -127,9 +125,5 @@ def get_sql_case_reindexer():
         pillow=get_sql_case_to_elasticsearch_pillow(),
         change_provider=SqlCaseChangeProvider(),
         elasticsearch=get_es_new(),
-        index_info=_get_case_index_info(),
+        index_info=CASE_INDEX_INFO,
     )
-
-
-def _get_case_index_info():
-    return get_index_info_from_pillow(CasePillow(online=False))

